python/tests-integration/test_signer.py

```diff
@@ -0,0 +1,166 @@
+import os
+import pathlib
+import shutil
+import subprocess
+import tempfile
+
+import pytest
+
+DATA_DIRECTORY = pathlib.Path(__file__).parents[2].absolute() / "data"
+
+_GPG_INSTRUCTIONS = """
+Key-Type: EDDSA
+  Key-Curve: ed25519
+Subkey-Type: ECDH
+  Subkey-Curve: cv25519
+Name-Real: Integration test key
+Expire-Date: 0
+%no-protection
+%commit
+"""
+
+
+@pytest.fixture
+def ephemeral_gpg_keys(tmp_path: pathlib.Path) -> None:
+    """Generate public and private GPG keys.
+
+    :yields: Tuple of paths to private and public GPG key.
+    """
+    # Create fresh GPG keys
+    instructions_file = tmp_path / "instructions"
+    instructions_file.write_text(_GPG_INSTRUCTIONS)
+    genkey = subprocess.run(
+        [
+            "gpg",
+            "--batch",
```

Comment:
Could you change the line `gpg` to `/usr/bin/gpg` so it would be consistent with explicit path usage from other files? Like it is already in `crypto.py`, `_keygen.py`, and `test_crypto.py`